### PR TITLE
MM-57913 Increase maximum length of Preferences.Value column (#27297)

### PR DESCRIPTION
**Pull request by hmhealey**
#### Summary
As discussed in the ticket, we're backporting https://github.com/mattermost/mattermost/pull/27297 to the 9.5 ESR

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-57913

#### Release Note
```release-note
Increased the maximum length of the Value column of the Preferences table
```

### CODE REVIEW

File: server/channels/db/migrations/migrations.list
Comment (hmhealey) [inline server/channels/db/migrations/migrations.list:1]: I don't think we usually backport things which require DB migrations, but it's needed for this one. I left the migration name/number the same as it is on master even though it introduces a gap between these. I hope the migration code will be able to pick up the ones in between 119 and 122 if someone updates to this 9.5.x release and then to a newer version

Comment (lieut-data) [reply server/channels/db/migrations/migrations.list:1]: I think so, but will defer to @isacikgoz to confirm.

**Review by lieut-data — APPROVED** [review]

File: server/channels/db/migrations/migrations.list
Comment (agnivade) [reply server/channels/db/migrations/migrations.list:1]: Yes you are right, the tool will fill the gap. This was on the features we needed to support cherry-picks to work seamlessly without changing anything.

**Review by agnivade — APPROVED** [review]

File: server/channels/db/migrations/migrations.list
Comment (hmhealey) [reply server/channels/db/migrations/migrations.list:1]: Awesome, thanks for the confirmation. The migration tool is so much better than the old way of doing migrations!

**Pull request merged**